jsAgent/tour.js

```
@@ -1,8 +1,332 @@
+     * Initializes the tour by asynchronously loading floating-ui libraries.
+     * @param {function(boolean): void} cb - Callback function, called with true on success, false on failure.
+     * @returns {void}
+     */
+    init : async function (cb) {
+        const result = window.bwonboarddata.tour[0];
+        const tourId = result.id;
+        const tourData = await bw.data.getTourById(tourId);
+        bw.data.getTourById(tourId).then((tourData) => {
+            console.log(tourData);
+            bw.tour.tourData = tourData;
+            cb && cb(tourData);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)